cosmwasm/enclaves/shared/cosmwasm-types/generic/src/lib.rs

```diff
@@ -73,15 +73,19 @@ impl BaseEnv {
         )
     }
 
-    pub fn into_versioned_env(self, api_version: &CosmWasmApiVersion) -> CwEnv {
+    pub fn into_versioned_env(
+        self,
+        api_version: &CosmWasmApiVersion,
+        is_empty_sender: bool,
+    ) -> CwEnv {
         match api_version {
-            CosmWasmApiVersion::V010 => self.into_v010(),
-            CosmWasmApiVersion::V1 => self.into_v1(),
+            CosmWasmApiVersion::V010 => self.into_v010(is_empty_sender),
+            CosmWasmApiVersion::V1 => self.into_v1(is_empty_sender),
```

Comment:
I'm not a big fan of complicating the interfaces with parameters like this. Why not just add a "set_sender" function and just call env.set_sender("")?

Reply:
Reverted in 4b8ddafe382deef5faa305f289d48cfe46e068ed